easybuild/easyconfigs/f/fastStructure/fastStructure-f94d4e53-intel-2014b.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'MakeCp'
+
+name = 'fastStructure'
+version = 'f94d4e53'
+
+homepage = 'http://rajanil.github.io/fastStructure/'
+description = """fastStructure is an algorithm for inferring population structure from large SNP genotype data. It is
+ based on a variational Bayesian framework for posterior inference and is written in Python2.x. """
+
+toolchain = {'name': 'intel', 'version': '2014b'}
+
+source_urls = ["https://github.com/rajanil/fastStructure/archive/"]
+sources = ['%(version)s.tar.gz']
+
+patches = ['shebang.patch']
```

Comment:
rename patch to include version it was patched against, something like `fastStructure-f94d4e53_fix-shebang.patch`